config/rbac/role.yaml

```diff
@@ -123,15 +123,3 @@ rules:
   - get
   - patch
   - update
-- apiGroups:
-  - ""
-  resources:
-  - services
```

Comment:
Don't we need these permission to maintain service?

Reply:
we need them
former rbac is generated from resoruceconsist's remarks, maybe generating from who uses adapters is better.
a new commit pushed, ptal.